currency_rate_update/services/update_service_VN_VCB.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+# © 2017 Binh Lam
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+"""Add service update VCB exchange rates."""
```

Comment:
Shouldn't this be just a comment using `# Add...`?

Reply:
It's a doc string.

Reply:
I think this is duplicated with line 18.

I think a docstring as the first line of the class is more appropriate so you should keep line 18 and remove this line.